### PR TITLE
Eng 3998: Updates doc based on new design for ds project details

### PR DESCRIPTION
**Pull request by bredamc**
Updates the doc based on new design for ds project details. 

Commits grouped as follows:

- [x] - DataConnection
- [x] - Model
- [x] - Permission
- [x] - Pipeline
- [x] - Project
- [x] - Storage
- [x] - Workbench
- [x] - Other

### CODE REVIEW

**Review by jbyrne-redhat — APPROVED** [review]
Changes in the `Project details redesign: Models` commit (69d0687) look good to me, @bredamc!

File: modules/starting-a-workbench.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 = Starting a workbench
 
 [role='_abstract']
-You can manually start a data science project's workbench from the *Details* page for the project. By default, workbenches start immediately after you create them.
+You can manually start a data science project's workbench from the project details page. By default, workbenches start immediately after you create them.
```
Comment (grainnejenningsRH) [inline modules/starting-a-workbench.adoc:7]: Looks like this functionality is on the Workbenches tab now, i.e.: 'You can manually start a data science project's workbench from the *Workbenches* tab for the project'.

**Review by grainnejenningsRH — CHANGES_REQUESTED** [review]

**Review by grainnejenningsRH — APPROVED** [review]
DSProject, Permission, and Workbench changes LGTM!

File: modules/updating-a-connected-data-source.adoc [hunk 2]
```diff
@@ -19,18 +19,19 @@ endif::[]
...
 . From the {productname-short} dashboard, click *Data Science Projects*.
 +
-The *Data science projects* page opens.
+The *Data Science Projects* page opens.
 . Click the name of the project whose data source you want to change.
 +
 A project details page opens.
-. Click the action menu (*&#8942;*) beside the data source that you want to change in the *Data connections* section and click *Change connected workbenches*.
+. Click the *Data connections* tab.
+. Click the action menu (*&#8942;*) beside the data source that you want to change and then click *Change connected workbenches*.
```
Comment (grainnejenningsRH) [inline modules/updating-a-connected-data-source.adoc:27]: Looks like this section needs to be:

'and then click *Edit data connection*.
+
The *Edit data connection* dialog opens.
. In the *Connected workbench* field, select a workbench to connect.
. Click *Update data connection*.'

File: modules/adding-a-data-connection-to-your-data-science-project.adoc [hunk 3]
```diff
@@ -37,8 +38,8 @@ The *Add data connection* dialog opens.
 . Click *Add data connection*.
 
 .Verification
-* The data connection that you added appears in the *Data connections* section on the *Details* page for the project.
-* If you selected a workbench, the data connection is visible in the *Workbenches* section on your data science project page.
+* The data connection that you added appears in the *Data connections* tab for the project.
+* If you selected a workbench, the data connection is visible in the *Workbenches* tab for the project.
```
Comment (grainnejenningsRH) [inline modules/adding-a-data-connection-to-your-data-science-project.adoc:42]: I'm not seeing data connections displayed on the Workbenches tab - I can see connected workbenches displayed under *Connected workbenches* for the data connection on the Data connections tab, though.

**Review by grainnejenningsRH — COMMENTED** [review]
Cluster storage changes LGTM.

File: modules/updating-a-connected-data-source.adoc (hunk 2 above)
Comment (bredamc) [reply modules/updating-a-connected-data-source.adoc]: Good catch, thank you! Updated now!

File: modules/adding-a-data-connection-to-your-data-science-project.adoc (hunk 3 above)
Comment (bredamc) [reply modules/adding-a-data-connection-to-your-data-science-project.adoc]: Another good catch, thank you! Updated now!

**Comment by bredamc** [comment]
@grainnejenningsRH Added two more commits to address pipelines and TrustyAI updates. Can you please review and approve? Many thanks :)

**Review by grainnejenningsRH — APPROVED** [review]

**Pull request merged**